packages/core/src/api/blockManipulation/commands/moveBlocks/moveBlocks.ts

```
@@ -142,13 +142,22 @@ function flattenColumns(
  * @param placement Whether to insert the blocks before or after the reference
  * block.
  */
+/** Find and fix broken columnLists after block operations */
+function ensureColumnLists(doc: any, tr: any) {
+  doc.nodesBetween(0, doc.content.size, (node: any, pos: number) => {
+    if (node.type.name === "columnList") {
+      try { fixColumnList(tr, pos); } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)